src/compiler/types.ts

```diff
@@ -2382,19 +2382,16 @@ namespace ts {
         Union                   = 1 << 19,  // Union (T | U)
         Intersection            = 1 << 20,  // Intersection (T & U)
         Anonymous               = 1 << 21,  // Anonymous
-        Instantiated            = 1 << 22,  // Instantiated anonymous type

```

Comment:
I only need one flag right now (Spread), but I'll need another for Rest and Ryan will need one for Subset. Maybe ObjectLiteral is a better target? 

FreshLiteral seems pretty easy to excise as well, but there are a lot of literals in any given program.